express-postgres/server.ts

```diff
@@ -0,0 +1,85 @@
+import pg from 'pg';
+import express from 'express';
+import { ClientError, errorMiddleware } from './lib/index.js';
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/pagila',
+  ssl: {
+    rejectUnauthorized: false,
+  },
+});
+
+const app = express();
+
+app.get('/api/films', async (req, res, next) => {
+  try {
+    const sql = `
+      select "title",
+             "filmId"
+      from "films"
+      order by "replacementCost" desc;
+    `;
+    const result = await db.query(sql);
+    if (!result) {
+      throw new ClientError(404, 'films not found');
+    }
+    res.send(result);
```

Comment:
This isn't quite right. This returns the entire result object, but you want to return just the films.